Wireframe/index.html

```diff
@@ -8,25 +8,41 @@
   </head>
   <body>
     <header>
-      <h1>Wireframe</h1>
+      <h1>From Wireframe to Code</h1>
       <p>
-        This is the default, provided code and no changes have been made yet.
+        From blueprint to browser: How wireframes, READMEs, and Git branches come together in a real project
       </p>
     </header>
     <main>
       <article>
-        <img src="placeholder.svg" alt="" />
-        <h2>Title</h2>
+        <img src="/Wireframe/bNEObvG_oAgqT83iZGoQR_vadb3Py6xbVh_Xy6vwEdM.webp" alt="" />
```

Comment:
🤔  Can you review the `<alt>` elements within your index.html file and see what you might be missing in your code?
https://www.w3schools.com/tags/att_img_alt.asp